server/meshmodel/relationships/peer_configurational_relationship.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
@MUzairS15, please confirm whether these new relationship definitions will be automatically accounted for on Meshery Server boot. If they will, then @Boombag0607, further scrutiny is required. Define manual tests and show the outputs of your results; e.g.

❌ A ConfigMap of one Kubernetes version (e.g. v1.25.0) can relate to a Service within a different Kubernetes version (e.g. v1.27.1).
✅ A ConfigMap of one Kubernetes version (e.g. v1.25.0) can relate to a Service within a different Kubernetes version (e.g. v1.25.0).

Reply:
@leecalcote I'll take a look if these definitions will sit well. Or we can focus on getting the MeshModal relationship doc approved first, then code them in the server.